tools/deb/istio-iptables.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
yes there are risks but I do think cross reference is useful - including highlighting the differences